### PR TITLE
Proposal Policy definition issue #325

### PR DESCRIPTION
**Pull request by felixfaassen**
A first proposal for adding "Policy" to the glossary see issue #325 (Term "Policy" or "Public Policy" )

-----
[View rendered glossary.md](https://github.com/publiccodenet/standard/blob/felixfaassen-policy-definition-325/glossary.md)

### CODE REVIEW

**Comment by bvhme** [comment]
Hey @ElenaFdR, I'm quite curious as to how you view this.

**Comment by bvhme** [comment]
Thanks @ElenaFdR, should that block this pull request or should we create a new issue?

File: glossary.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ Open source is defined by the Open Source Initiative in their [Open Source Defin
 
 An open standard is any standard that meets the Open Source Initiative's [Open Standard Requirements](https://opensource.org/osr).
 
+## Policy
+
+A policy is a deliberate system of principles to guide decisions and achieve rational outcomes. A policy is a statement of intent, and is implemented as a procedure or protocol. Policies are generally adopted by a governance body within an organization. Policies can assist in both subjective and objective decision making. Public policy is the process by which governments translate their political vision into programmes and actions to deliver outcomes. In the Standard the word 'policy' refers to policy created and adopted by public organizations such as governments and municipalities.
```
Comment (bvhme) [inline glossary.md:33]: I suggest breaking this up into multiple paragraphs as paragraphs this long can be hard to read. (I've added [semantic line breaks](https://sembr.org/) in my suggestion for easier version control)

```suggestion
A policy is a deliberate system of principles to guide decisions and achieve rational outcomes.
A policy is a statement of intent, and is implemented as a procedure or protocol.
Policies are generally adopted by a governance body within an organization.
Policies can assist in both subjective and objective decision making.

Public policy is the process by which governments translate their political vision into programmes and actions to deliver outcomes.

In the Standard the word 'policy' refers to policy created and adopted by public organizations such as governments and municipalities.
```

Comment (felixfaassen) [reply glossary.md:33]: Nice addition @bvhme . @ElenaFdR may I suggest to pull this in and make a seperate issue for the points you mentioned ""At the national level, policy and legislation (the law) are usually separate. The distinction is often more blurred in local government."?

File: glossary.md [hunk 2]
```diff
@@ -28,6 +28,17 @@ Open source is defined by the Open Source Initiative in their [Open Source Defin
...
 An open standard is any standard that meets the Open Source Initiative's [Open Standard Requirements](https://opensource.org/osr).
 
+## Policy
+
+A policy is a deliberate system of principles to guide decisions and achieve rational outcomes.
+A policy is a statement of intent, and is implemented as a procedure or protocol.
+Policies are generally adopted by a governance body within an organization.
+Policies can assist in both subjective and objective decision making.
+
+Public policy is the process by which governments translate their political vision into programmes and actions to deliver outcomes.
```
Comment (ElenaFdR) [inline glossary.md:38]:
```suggestion
Public policy is the process by which governments translate their political vision into programmes and actions to deliver outcomes.

At the national level, policy and legislation (the law) are usually separate. The distinction is often more blurred in local government.
```

**Comment by felixfaassen** [comment]
@bvhme This build was failing because:

./_site/criteria/bundle-policy-and-code.html
  *  External link https://www.bpmnquickguide.com/view-bpmn-quick-guide/ failed: response code 0 means something's wrong.
             It's possible libcurl couldn't connect to the server or perhaps the request timed out.
             Sometimes, making too many requests at once also breaks things.
             Either way, the return message (if any) from the server is: Peer certificate cannot be authenticated with given CA certificates
htmlproofer 3.15.3 | Error:  HTML-Proofer found 1 failure!
The command "./script/cibuild.sh" exited with 1.

I added a config line to HTML proofer to disable external SSL host/cert checking which can break builds when either the external host uses outdate SSL certs OR external hosts use SSL certs our travis container cannot verify as it might be missing the latest CA certs. 

See test.sh for changes (and commit log). Additional solution could be to add an extra command to the .travis_yml by having the certs updated each time travis spins up a container but this requires root priveles meaning builds will go much slower. 

this is the new line:
bundle exec htmlproofer --url-ignore "/github.com/(.*)/edit/" ./_site --typhoeus-config '{"ssl_verifypeer": false, "ssl_verifyhost":0}'

**Comment by felixfaassen** [comment]
Merge  including additional config settings in test.sh to prevent builds failing when htmlproofer cannot verify external hosts SLL certs

**Comment by bvhme** [comment]
That sounds like a fix for https://github.com/publiccodenet/standard/issues/338

Why close this pull request? Is the content in it not needed anymore?

**Comment by bvhme** [comment]
I would prefer a separate pull request for that addition from this content one, so that we can have those discussions separately and can merge in that fix as quickly as possible without having to agree on the content as well and vice versa.

**Comment by felixfaassen** [comment]
> I would prefer a separate pull request for that addition from this content one, so that we can have those discussions separately and can merge in that fix as quickly as possible without having to agree on the content as well and vice versa.

I agree, I was under the impression you and Elana already approved the content by reviewing my pull request. The only thing standing in the way of pulling it in was this error. I was not aware of the separate issue you created. 

But great that it works now right? Took some time to figure this out ;-)

**Comment by bvhme** [comment]
Elena hasn't re-reviewed it, and thus not approved, but perhaps you have just incorporated her changes so we can take that as an implicit approval?

Regardless I think this is a great fix and well documented, and I think it is worth merging regardless of that conversation.

**Comment by bvhme** [comment]
Still this pull request is unmerged and closed, which seems like the wrong state regardless. Do we not want to merge both?

**Comment by felixfaassen** [comment]
@bvhme sorry my bad for not merging it.  I would propose merge it both. and I will take note to have this stuff as seperate issues in the future

**Comment by bvhme** [comment]
Thanks Felix :)

We're writing the documentation of the history as well so having pull requests make sense is important. However, I'm happy this landed.

**Comment by felixfaassen** [comment]
Cool nice to hear Boris ;-) That is a great guideline (we are writing history) so it better be good ;-) I'll keep that in mind!

**Pull request merged**